test/lib/build.test.js

```diff
@@ -478,22 +485,6 @@ describe('Build Model', () => {
         });
     });
 
-    describe('user', () => {
```

Comment:
@ian-fox the code for this test was removed (see https://github.com/screwdriver-cd/models/pull/180/files/fe15f0990dc79a91e700d29a1d7789399ebf4aa9#diff-26b681bd4dac10f25d5293c7595c435b)